### PR TITLE
test(ci): add Node.js 11 to CI test matrix

### PR DESCRIPTION
**Pull request by watson**

### CODE REVIEW

**Comment by Qard** [comment]
I made a fix for the current error in this PR at #634. The api schema tests and hapi tests are also currently failing on Node.js 11 though.

**Comment by watson** [comment]
@Qard I've rebased with master. I think this leaves us only with the hapi test-failures

File: .travis.yml [hunk 1]
```diff
@@ -39,6 +40,9 @@ jobs:
     ###########################################
 
     # Disable Async Hooks
```
Comment (watson) [inline .travis.yml]: Test, please ignore me:

```suggestion
    # Disable Async-Hooks
```

Comment (watson) [reply .travis.yml]: Made this review to help github staff fix the a bug in the suggested change feature - just ignore

**Pull request merged**